### ISSUE TITLE
Uncaught exception 'Exception' with message 'Editable section not found.'

### ISSUE THREAD

**Issue by pa-de-solminihac** [versionpress/versionpress#999]
When  I try to install VersionPress 3.0-rc it fails : 

```
PHP Fatal error:  Uncaught exception 'Exception' with message 'Editable section not found.' in /path/to/www/blog/wp-content/plugins/versionpress/src/Utils/WpConfigEditor.php:74
Stack trace:
#0 /path/to/www/blog/wp-content/plugins/versionpress/src/Utils/WpConfigEditor.php(37): VersionPress\\Utils\\WpConfigEditor->updateConfig('master', '/^(\\\\s*define\\\\s*...', 'define('VP_ENVI...', false)
#1 /path/to/www/blog/wp-content/plugins/versionpress/admin/inc/activate.php(98): VersionPress\\Utils\\WpConfigEditor->updateConfigConstant('VP_ENVIRONMENT', 'master')
#2 /path/to/www/blog/wp-content/plugins/versionpress/admin/index.php(12): require_once('/path/to/...')
#3 /path/to/www/blog/wp-admin/admin.php(264): include('/path/to/...')
#4 {main}
  thrown in /path/to/www/blog/wp-content/plugins/versionpress/src/Utils/WpConfigEditor.php on line 74, referer: http://domain.com/blog/wp-admin/admin.php?page=versionpress
```

I disabled all external plugins.
I just have one warning  : Web size. The initialization will take a little longer. This website contains 871 entities. 

OS : debian jessie
PHP : 5.6.19
VP : 3.0-rc
WP : 4.5–fr_FR

**Comment by pavelevap** [comment]
Hi, it seems to me that there is some problem with your `wp-config.php` file. Is this file writable? Are there any special changes (added constants)?

**Comment by JanVoracek** [comment]
Hi. We use comment `/* That's all, stop editing! Happy blogging. */` in the original `wp-config.php` to detect the position for adding new constants. We hoped users will keep this comment in the `wp-config`. Now we see we were wrong. We have to find out some better solution.

**Comment by borekb** [comment]
@pa-de-solminihac Can you share your `wp-config.php` (with all secrets removed) please?

**Comment by pa-de-solminihac** [comment]
My `wp-config.php` file has been modified by some extensions. I disabled all extensions, I have cleaned my `wp-config.php`, it is now the exact copy of `wp-config-sample.php`, with secrets replaced.

_chmod_ is `640`, _encoding_ is as follows:
```vim
fileformat=unix
encoding=utf-8
fileencoding=utf-8
nobomb
```

Still doesn't work, though, with the same error message.

**Comment by borekb** [comment]
Just to be sure, could you please upload the current `wp-config.php` (secrets removed)?

**Comment by pa-de-solminihac** [comment]
Of course, here it is: https://gist.github.com/pa-de-solminihac/c980048ac93ce34b78929785947ba3bf

**Comment by borekb** [comment]
Oh, we didn't realize comments in `wp-config.php` are localized. Oops!

I'll create a new issue to improve our "Happy blogging" detection. You can expect the fix to be part of 3.0. Thanks for uncovering this issue!

```
/* C'est tout, ne touchez pas à ce qui suit ! Bon blogging ! */
```

**Comment by pa-de-solminihac** [comment]
Ok I based my `wp-config.php` file on a fresh non localized `wp-config-sample.php` file and install finished.

Thanks. Let's play now!
